client/package.json

```diff
@@ -6,7 +6,9 @@
   },
   "scripts": {
     "clean": "rimraf output",
-    "build": "spago bundle-app --path $npm_package_config_configpath --purs-args '--censor-lib --strict' --to public/js/index.js"
+    "test": "spago test --path $npm_package_config_configpath",
+    "build": "spago build --path $npm_package_config_configpath",
```

Comment:
Could we switch this environment variable to something a little shorter? Perhaps `TRY_CONFIG`?

Reply:
The reason we have the `npm_package_config_` prefix is so that it can pick up the values from the `config` key in package.json by default if they haven't been explicitly overridden (e.g. like we do in CI: https://github.com/purescript/trypurescript/blob/52c81fc168d2e39f701ed2aaa61694147f6024bc/ci/build.sh#L16). We have to include the `npm_package_config_` prefix for this to work. If there's another way to make `npm run build` Just Work in development and for the production config to be used in CI I'm happy to consider that.